src/keyAPI.js

```diff
@@ -0,0 +1 @@
+export default ('643484ca-9138-4647-b058-430c60bdb96f')
```

Comment:
We can try to do that, i just don't know what it involves and don't feel it necessary to incorporate the .env since frontend alone doesn't typically have to hide keys. that's usually something on backend so i'm not super worried about learning how to do it.